R/vulnerability/disasters-emergencies/flooding/2022-interim/england/metadata.md

```diff
@@ -0,0 +1,31 @@
+## England Flooding Vulnerability (using 2022 interim data) Metadata
+
+| Domain | Indicator | Year | Source | License | Indicator Code Added to `R/` | Data Added to `data/` |
+| --- | --- | --- | --- | --- | --- | --- |
+| Age | Young children (% under 5 years) | 2021 | ONS/geographr | :x: | :heavy_check_mark: | :heavy_check_mark: |
+| Age | Older people (% 75+ years) | 2021 | ONS/geographr | :x: | :heavy_check_mark: | :heavy_check_mark: |
```

Comment:
Think the population data from geographr is as at 2019 based on https://github.com/humaniverse/geographr/blob/main/data-raw/query-urls.R. 
```suggestion
| Age | Young children (% under 5 years) | 2019 | ONS/geographr | :x: | :heavy_check_mark: | :heavy_check_mark: |
| Age | Older people (% 75+ years) | 2019 | ONS/geographr | :x: | :heavy_check_mark: | :heavy_check_mark: |
```